twitter.html

```diff
@@ -0,0 +1,333 @@
+          <div class="e1-b-el-1">
+            <img
+              src="/images/pitures/pexels-markus-winkler-4160060.jpg"
+              alt="picture"
+              class="pic-profile-1 position"
+            />
+          </div>
+          <div class="e1-b-e2-2 position">
+            <ion-icon name="language-outline"></ion-icon
+            ><ion-icon name="language-outline"></ion-icon
+            ><ion-icon name="language-outline"></ion-icon>
+            <br />
+            <p>@account_name</p>
+          </div>
+          <div class="e1-b-e2-3"><p class="dots position">...</p></div>
+        </div>
+      </section>
+
+      <div class="head-n-main">
+        <!--HEADER   -->
+        <header class="header hdr">
+          <div class="home-n-str">
+            <div>
+              <p class="font-home-list-whats-h home">Home</p>
+            </div>
+          </div>
+          <div></div>
+          <div class="star">
+            <ion-icon name="sparkles-outline" class="house-2"></ion-icon>
+          </div>
+          <div class="img-header">
+            <img
+              src="images/pitures/pexels-markus-winkler-4160060.jpg "
+              alt="Picture"
+              class="pic-profile-2"
+            />
+          </div>
+          <div class="whats-h-header">
+            <p><strong>What's happening?</strong></p>
+          </div>
+          <div></div>
+          <div></div>
+          <div class="header-icons">
+            <ion-icon name="image-outline" class="house"></ion-icon>
+            <ion-icon name="gift-outline" class="house"></ion-icon>
+            <ion-icon name="bar-chart-outline" class="house"></ion-icon>
+            <ion-icon name="happy-outline" class="house"></ion-icon>
+            <ion-icon name="calendar-clear-outline" class="house"></ion-icon>
+          </div>
+          <!-- twitter button header -->
+          <button class="header-btn">Tweet</button>
+        </header>
+
+        <!-- MAIN  -->
+
+        <main class="main">
+          <div>
+            <img
+              src="images/pitures/pexels-markus-winkler-4160060.jpg"
+              class="pic-profile-2"
+              alt="Picture"
+            />
+          </div>
+          <p class="firt-tweet">
+            <ion-icon name="tennisball-outline"></ion-icon>
+            <span class="gry-font">
+              Lorem ipsum dolor sit amet consectetu
+            </span>
+            <br />
+            <strong>Lorem ipsum dolor</strong
+            ><span class="gry-font-big">@consectetu 26m</span> <br />Lorem ipsum
+            dolor sit amet consectetur, adipisicing elit. Officia, doloribus? Nu
+            <ion-icon name="stats-chart-outline" class="ylw-clr"></ion-icon>
+            <br /><br />
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti
+            nihil consequatur dolorum tenetur fuga
+            <ion-icon name="storefront-outline" class="clr-grn"></ion-icon>
+            <br />
```

Comment:
We typically prefer not to use `<br />` tags to add vertical spacing to our webpage. Instead, it is better to add vertical spacing with `margin-bottom`.